### PR TITLE
Get Event Endpoint

### LINKED ISSUE #260: Create an Endpoint to Get An Event
Below you can find the details:

| path                               | type   | functionality                               | parameters                                                                                                                                                                                                                | example                                                                                                                                                                                                                                  |
|------------------------------------|--------|---------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| events/<event_id>/                             | GET | get event information         |   |  |

Example:
```
{
  "@context": "https://schema.org",
  "@type": "SportsEvent",
  "name": "Let's Play Football",
  "sport": "football",
  "duration": 5,
  "startDate": "2015-07-05T15:30-07:00",
  "location": {
      "@context": "https://schema.org",
      "@type": "Place",
      "geo": {
        "@type": "GeoCoordinates",
        "latitude": "40.75",
        "longitude": "73.98"
      },
      "address": "Beşiktaş, İstanbul"
  },
  "maximumAttendeeCapacity": 10,
    "attendee": [
  	{
      "@context": "https://schema.org",
      "@type": "Person",
      "@id":5 
    },
    {
      "@context": "https://schema.org",
      "@type": "Person",
      "@id":4 
    }
  ],
  "organizer":   {
          "@context": "https://schema.org",
          "@type": "Person",
          "@id":3 
    },
  "description": "Let's play football on this beautiful day",
  "additionalProperty": [
    {
      "@type": "PropertyValue",
      "name": "minimumAttendeeCapacity",
      "value": 2
    },
    {
      "@type": "PropertyValue",
      "name": "maxSpectatorCapacity",
      "value": 40
    },
    {
      "@type": "PropertyValue",
      "name": "spectator",
      "value":[
        {
          "@context": "https://schema.org",
          "@type": "Person",
          "@id":2 
        }
      ]
    }
  ]
}

```


a function in the event class should created and it should be called by the endpoint function.
Unittests should be written.

We should choose a schema for our events. I think we can go with [SportsEvent](https://schema.org/SportsEvent) schema. But there is not a field about spectators. I added an example with the idea I have. I think we can add the minimumAttendeeCapacity, maxSpectatorCapacity and spectator as additionalProperty. What do you think? If you agree, I will construct event database class with these fields.

### PR DESCRIPTION
**Pull request by berkatil**
Closes #260

### CODE REVIEW

File: back-end/sports_platform/sports_platform_api/models/event_models.py [hunk 1]
```diff
@@ -65,6 +95,52 @@ def create_event(data):
...
+             },{
+            "@type": "PropertyValue",
+            "name": "maxSpectatorCapacity",
+            "value": self.maxSpectatorCapacity
+             },{
+            "@type":"PropertyValue",
+            "name":"spectator",
+            "value":[{"@context":"https://schema.org","@type":"Person","@id": spectator.user} for spectator in spectators]
+            }
+        ]
```
Comment (fileames) [inline back-end/sports_platform/sports_platform_api/models/event_models.py:129]: For the spectators, I think we can use the audience field without additionalProperty.

Comment (berkatil) [reply back-end/sports_platform/sports_platform_api/models/event_models.py:125]: thanks, I have changed

File: back-end/sports_platform/sports_platform_api/models/event_models.py [hunk 2]
```diff
@@ -65,6 +95,49 @@ def create_event(data):
...
+        return [
+            {
+             "@type": "PropertyValue",
+             "name": "minimumAttendeeCapacity",
+             "value": self.minimumAttendeeCapacity
+             },{
+            "@type": "PropertyValue",
+            "name": "maxSpectatorCapacity",
+            "value": self.maxSpectatorCapacity
+             }
```
Comment (fileames) [inline back-end/sports_platform/sports_platform_api/models/event_models.py:124]: Also interesteds should be added as additionalProperty 
```
{
    "@type": "PropertyValue",
    "name": "interesteds",
    "value": list of users
}
```

Comment (berkatil) [reply back-end/sports_platform/sports_platform_api/models/event_models.py:129]: thanks, it is added

File: back-end/sports_platform/sports_platform_api/models/event_models.py [hunk 3]
```diff
@@ -65,6 +95,55 @@ def create_event(data):
...
+             },{
+            "@type": "PropertyValue",
+            "name": "maxSpectatorCapacity",
+            "value": self.maxSpectatorCapacity
+             },
+             {
+            "@type": "PropertyValue",
+            "name": "interesteds",
+            "value": [{"@context":"https://schema.org","@type":"Person","@id": interested.user.user_id, "identifier": interested.user.identifier} for interested in interesteds]
+             }
```
Comment (fileames) [inline back-end/sports_platform/sports_platform_api/models/event_models.py:129]: I think acceptWithoutApproval should also be added as additionalProperty.

**Pull request merged**